workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/IReader.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using Improbable.Worker;
-using Unity.Entities;
 
-namespace Improbable.Gdk.Core.MonoBehaviours
```

Comment:
Why are we moving everything into just `Core`

Reply:
Plan is to move everything out in a separate PR before merging into master as far as I understand (right now existing files are in just Core)

Reply:
https://github.com/spatialos/UnityGDK/pull/141